parsec/core/remote_devices_manager.py

```diff
@@ -21,148 +22,129 @@
 
 DEFAULT_CACHE_VALIDITY = 60 * 60  # 3600 seconds, 1 hour
```

Comment:
Can we use the constant `REMOTE_DEVICE_MANAGER_CACHE_VALIDITY` defined in `remote_device_manager.rs` ?

Reply:
It's weird to use the const of `RsRemoteDevicesManager` for `PyRemoteDevicesManager`

Reply:
Depend on how you see it, if they don't share the same value the tests would fail.

It would be simpler to use the same value between python & rust.

Reply:
And it's a property from RsRemoteDevicesManager, so it needs self (not really) but we need the compatibility